tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -105,10 +105,10 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
       val jwt2 = JwtHelper.generateValidJwt(secondaryTestUserId.getId)
       val deviceName = Try(DeviceName.validate("myDevice")).toEither.value
       val newDeviceResult = Try(
-        IronOxide.generateNewDevice(jwt, primaryTestUserPassword, DeviceCreateOpts.create(deviceName.clone), null)
+        IronOxide.generateNewDevice(jwt, primaryTestUserPassword, new DeviceCreateOpts(deviceName.clone), null)
```

Comment:
Shouldn't these clone calls be unnecessary since the options have been fixed?

Reply:
Good catch. I thought I had gotten these in the other PR, but apparently I forgot.
Removed.